algorithms/scaling/scaler.py

```diff
@@ -790,14 +793,23 @@ def round_of_outlier_rejection(self):
         """Perform a round of outlier rejection, set a new outliers array."""
         assert self.global_Ih_table is not None
         if self.params.scaling_options.outlier_rejection:
+            Esq_outlier_indices = determine_Esq_outlier_index_arrays(
+                self.global_Ih_table, self.experiment, self.params.scaling_options.emax
+            )[0]
             outlier_indices = determine_outlier_index_arrays(
                 self.global_Ih_table,
                 self.params.scaling_options.outlier_rejection,
                 self.params.scaling_options.outlier_zmax,
             )[0]
             self.outliers = flex.bool(self.n_suitable_refl, False)
             self.outliers.set_selected(outlier_indices, True)
+            self.outliers.set_selected(Esq_outlier_indices, True)
             if self._free_Ih_table:
+                Esq_outlier_indices = determine_Esq_outlier_index_arrays(
```

Comment:
These particular outlier indices don't look to be used?

Reply:
Good spot, there was a line of code missing in here.